source/math/FloatPoint2DVectorList.ooc

```diff
@@ -69,6 +69,18 @@ FloatPoint2DVectorList: class extends VectorList<FloatPoint2D> {
 		sortedY free()
 		result
 	}
+	getMean: func (indices: VectorList<Int>) -> FloatPoint2D {
+		result := FloatPoint2D new()
+		if (indices count > 0) {
+			for (i in 0 .. indices count) {
+				element := this[indices[i]]
```

Comment:
And it should probably be coded like 
```
buffer := this buffer pointer
element := buffer[i] as FloatPoint2D
```
(or something) to avoid calling `operator []`